include/bluetooth/mesh/dk_prov.rst

```diff
@@ -3,55 +3,51 @@
 Bluetooth Mesh provisioning handler for Nordic DKs
 ##################################################
 
-This application-side module is a basic implementation of provisioning process handling for Development Kits from Nordic Semiconductor.
+This application-side module is a basic implementation of the provisioning process handling for Development Kits from Nordic Semiconductor.
 It supports four types of Out Of Band (OOB) authentication methods and uses the Hardware Information driver to generate a deterministic UUID to uniquely represent the device.
+For more information about provisioning in Bluetooth Mesh, see the :ref:`zephyr:bluetooth_mesh_provisioning` page in Zephyr.
 
-Used primarily in sample applications, this handler acts as a reference implementation for the application-specific part of provisioning.
+Used primarily in :ref:`Bluetooth Mesh sample applications <ble_samples>`, this handler acts as a reference implementation for the application-specific part of provisioning.
```

Comment:
@trond-snekvik , how is the module configured to work with the samples? Is it automatically enabled? The sample Testing section focus on the app, and the provisioner handler is only briefly mentioned.

Reply:
It is enabled through the `CONFIG_DK_PROV` option and through calling `bt_mesh_dk_prov_init()` in main.